en/docs/learn/consume-api/invoke-apis/invoke-apis-using-tools/try-out-using-postman.md

```diff
@@ -0,0 +1,57 @@
+# Try out using Postman.
+
+Let`s see how to download a postman collection.
+
+1.  Sign in to the WSO2 Developer Portal (`https://<hostname>:9443/devportal`) and click an API (e.g. `Pizza` ) to go to the API overview.
+
+    ![({{base_path}}/assets/attachments/postman/postman_api_overview.png)]({{base_path}}/assets/attachments/postman/postman_api_overview.png)
+
+
+2.  Click `Try Out`to go to the Try out section.
```

Comment:
"Click `Try Out` " should be  as follows:

Click **Try Out**

Make sure to bold control objects like button names.